rest/route/project_test.go

```diff
@@ -470,12 +470,6 @@ func (s *ProjectPatchByIDSuite) TestRotateAndDeleteProjectPodSecret() {
 	s.NotZero(dbProjRef.ContainerSecrets[0].ExternalName)
 	s.NotZero(dbProjRef.ContainerSecrets[0].ExternalID)
 
-	externalID := dbProjRef.ContainerSecrets[0].ExternalID
-	s.Require().NotNil(h.vault)
-	initialStoredValue, err := h.vault.GetValue(ctx, externalID)
-	s.Require().NoError(err)
-	s.NotZero(initialStoredValue)
-
```

Comment:
@Kimchelly 

Removing the vault from the route handler means we can't test this anymore. Is this test safe to remove?